spec/helper.js

```diff
@@ -1,26 +1,37 @@
 import React from 'react';
 import chai from 'chai';
 import spy from 'chai-spies';
-import { mount, shallow } from 'enzyme';
-const jsdom = require('jsdom').jsdom;
+import Enzyme, { mount, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15';
+import { JSDOM } from 'jsdom';
+
+Enzyme.configure({
+  adapter: new Adapter()
+});
 
 chai.use(spy);
 
+const jsdom = new JSDOM('<!doctype html><html><body></body></html>');
+const { window } = jsdom;
+const document = window.document;
+
 global.React = React;
 global.expect = chai.expect;
 global.spy = chai.spy;
 global.mount = mount;
 global.shallow = shallow;
 
-global.document = jsdom('');
-global.window = document.defaultView;
-
-Object.keys(document.defaultView).forEach((property) => {
-  if (typeof global[property] === 'undefined' && property !== 'XMLHttpRequest') {
-    global[property] = document.defaultView[property];
-  }
-});
-
+global.window = window;
+global.document = window.document;
 global.navigator = {
   userAgent: 'node.js'
 };
+
+function copyProps(src, target) {
```

Comment:
How does this differ from Object.assign()?

Reply:
I'm not sure if it is... It's basically copied from the enzyme with jsdom docs: https://github.com/airbnb/enzyme/blob/master/docs/guides/jsdom.md

My guess is that it's a polyfill version of it since they don't know what node versions people are using. Maybe older versions of node don't support Object.assign(). That's all just me speculating though.